### ISSUE TITLE
Deno unstable feature as reported as `unsupported`

### ISSUE THREAD

**Issue by JOTSR** [unjs/runtime-compat#103]
### Describe the change

All `deno` unstable features as repported as unsupported (like: `GPUDeviceLostInfo`, `BroadcastChannel`).
Maybe a "partial" support state can be added like on mdn ([example](https://developer.mozilla.org/en-US/docs/Web/API/GPUDeviceLostInfo#browser_compatibility)).

### Additional information

- [X] Would you be willing to help?

**Comment by ascorbic** [comment]
Yeah right now we're just reporting things that are enabled by default in the latest version. It would be nice to track finer detail and versioning, like caniuse does.
